docs/content_management/preview_content_items.md

```diff
@@ -0,0 +1,32 @@
+---
+description: Preview content items of various types in many places of the Back Office.
+---
+
+# Preview Content items
+
+There are several places where you can see how the Content item could look to an end-user.
+The system behaves differently depending on whether you are in [Focus mode](../getting_started/discover_ui.md#focus-mode) or not and whether you selected a specific site from the **Site context** drop-down list in the top bar.
+
+- When you [edit a Page](create_edit_content_items.md), the editor provides a WYSIWYG experience.
```

Comment:
Please see comment from @juskora: "**WYSIWG is not an experience**, it's a kind of UI."
You are still writing "WYSIWYG experience".